src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -232,6 +235,26 @@ mutable struct PairedExplicitRK2Integrator{RealT <: Real, uType, Params, Sol, F,
     k_higher::uType
 end
 
+"""
+    calculate_cfl(ode_algorithm::AbstractPairedExplicitRKSingle, ode)
+
+This function computes the CFL number once using the initial condition of the problem and the optimal timestep (`dt_opt`) from the ODE algorithm.
+"""
```

Comment:
I like that you added a docstring. In that case, we should also export this function.

Reply:
I see. However, Michael advised against this some time ago in [PR](https://github.com/trixi-framework/Trixi.jl/pull/1908#discussion_r1578820114) for PERK2. But I docstringed this function since I see that the user should also be able to use it (the way that integrator algorithm also has its own docstring). But if you think its more appropriate to export this now, I can also edit this very quickly.